src/sentry/workflow_engine/migration_helpers/issue_alert_conditions.py

```diff
@@ -49,7 +49,7 @@ def create_every_event_data_condition(
         type=Condition.EVERY_EVENT,
         comparison=True,
         condition_result=True,
-        condition_group=dcg,
+        condition_group_id=dcg.id,
```

Comment:
for some reason the migration triggers an error when passing the `DataConditionGroup` object and creating an object here. using the id works